src/Rings.jl

```diff
@@ -212,6 +212,28 @@ rings are also called zero rings.
 """
 is_trivial(R::NCRing) = !is_domain_type(elem_type(R)) && iszero(one(R))
 
+@doc raw"""
+    is_zero(R::NCRing)
+
+Test whether the ring $R$ is trivial. However, the recommended method is is_trivial. A ring is trivial if it consists
+of a single element, or equivalently if its characteristic is 1. Such
+rings are also called zero rings.
```

Comment:
```suggestion
Test whether the ring $R$ is trivial. However, the recommended method
for doing this is [`is_trivial`](@ref).
```
I would not bother giving the definition here, users should look at `is_trivial` for that